src/pages/EditMonitor.vue

```diff
@@ -246,19 +249,66 @@
                             </template>
 
                             <!-- Hostname -->
-                            <!-- TCP Port / Ping / DNS / Steam / MQTT / Radius / Tailscale Ping only -->
-                            <div v-if="monitor.type === 'port' || monitor.type === 'ping' || monitor.type === 'dns' || monitor.type === 'steam' || monitor.type === 'gamedig' ||monitor.type === 'mqtt' || monitor.type === 'radius' || monitor.type === 'tailscale-ping'" class="my-3">
+                            <!-- TCP Port / Ping / DNS / Steam / MQTT / Radius / Tailscale Ping / SNMP only -->
+                            <div v-if="monitor.type === 'port' || monitor.type === 'ping' || monitor.type === 'dns' || monitor.type === 'steam' || monitor.type === 'gamedig' || monitor.type === 'mqtt' || monitor.type === 'radius' || monitor.type === 'tailscale-ping' || monitor.type === 'snmp'" class="my-3">
                                 <label for="hostname" class="form-label">{{ $t("Hostname") }}</label>
                                 <input id="hostname" v-model="monitor.hostname" type="text" class="form-control" :pattern="`${monitor.type === 'mqtt' ? mqttIpOrHostnameRegexPattern : ipOrHostnameRegexPattern}`" required>
                             </div>
 
                             <!-- Port -->
-                            <!-- For TCP Port / Steam / MQTT / Radius Type -->
-                            <div v-if="monitor.type === 'port' || monitor.type === 'steam' || monitor.type === 'gamedig' || monitor.type === 'mqtt' || monitor.type === 'radius'" class="my-3">
+                            <!-- For TCP Port / Steam / MQTT / Radius Type / SNMP -->
+                            <div v-if="monitor.type === 'port' || monitor.type === 'steam' || monitor.type === 'gamedig' || monitor.type === 'mqtt' || monitor.type === 'radius' || monitor.type === 'snmp'" class="my-3">
                                 <label for="port" class="form-label">{{ $t("Port") }}</label>
                                 <input id="port" v-model="monitor.port" type="number" class="form-control" required min="0" max="65535" step="1">
                             </div>
 
+                            <!-- SNMP Monitor Type -->
+                            <div v-if="monitor.type === 'snmp'" class="my-3">
+                                <label for="snmp_community_string" class="form-label">{{ $t("Community String") }}</label>
```

Comment:
How self-explaiatory is this field?
Is a helptext how a `Community String` can be gathered helpfull?

Reply:
I started helptexting ALL THE THINGS but is that really necessary? I don't see other monitors spelled out in this much detail.
![image](https://github.com/louislam/uptime-kuma/assets/9312603/6ac64753-3362-49f6-b073-8571ef9876a6)

Reply:
I think you understood me wrong.
I was asking IF the field is self-explaiatory.
I was not asking for a helptext, but was probing is here one might be beneficial.

I agree with your thought that these helptexts are a bit too wordy.
For example `condition`/`control value` from my point of view would not need a heptext.

Reply:
Okay, I removed helptexts from `SNMP Version` and `Control Value`, and rephrased the ones for `OID` and `Community String` to be more concise. :)
<img width="362" alt="image" src="https://github.com/louislam/uptime-kuma/assets/9312603/fb51a845-4990-4006-b80d-47da647d68b4">

Resolved with 2015142b00182c7d172c505b95b7c8aebddf43d7